SPECS/azurelinux-rpm-macros/macros.ocaml-srpm

```diff
@@ -13,3 +13,5 @@
 # This was removed in OCaml 4.09.
 # https://github.com/ocaml/ocaml/pull/2314
 %ocaml_native_profiling %{nil}
+
+%ocamldir %{_libdir}/ocaml
```

Comment:
This would be a good time to remove this macros file from this catch-all package and into its own `ocaml-srpm-macros` package. It looks like it's a very simple package without any BRs - Fedora's version contains only the `macros.ocaml-srpm` file for instance. With a separate package, it's going to be easier to track any future updates. The package would still need to land in the toolchain and the worker chroot but in case of such macros packages it's acceptable, since we're not causing any real bloat - the source of the macros file just moves from one package to another.

Reply:
Moved it